parsl/app/futures.py

```diff
@@ -56,8 +72,18 @@ def __init__(self, fut: Future, file_obj: File, tid: int) -> None:
         self._tid = tid
         self.file_obj = file_obj
         self.parent = fut
-
+        if app_fut:
```

Comment:
This seems like a bad idea. `self.app_fut` is either the optional kwarg, or the `fut`?

Reply:
This is now outdated, the `app_fut` keyword has been removed